cli/main.rs

```diff
@@ -1261,8 +1261,26 @@ pub fn main() {
     std::process::exit(1);
   }
 
-  let flags =
-    unwrap_or_exit(flags::flags_from_vec(args).map_err(AnyError::from));
+  let flags = match flags::flags_from_vec(args) {
+    Ok(flags) => flags,
+    Err(clap::Error {
+      kind,
+      message,
+      info,
+    }) if kind == clap::ErrorKind::HelpDisplayed
+      || kind == clap::ErrorKind::VersionDisplayed =>
+    {
+      clap::Error {
+        info,
+        kind,
+        message,
+      }
```

Comment:
Are you doing this de- and restructuring to get around "value moved" errors? You can pattern match on `Err(ref err)` or `Err(err @ clap::Error { .. })`, depending on what you want to do (but probably the former.)

Reply:
Oh, I didn't know that syntax! I'll update it.